app/code/Magento/Catalog/Model/ProductRepository.php

```diff
@@ -735,6 +737,7 @@ private function getCollectionProcessor()
     {
         if (!$this->collectionProcessor) {
             $this->collectionProcessor = \Magento\Framework\App\ObjectManager::getInstance()->get(
+                // phpstan:ignore
```

Comment:
Please add comment why you ignore phpStan on this line like  `// phpstan:ignore "Method Magento\TestModule\TestClass::testMethod() invoked with 1 parameter, 0 required."`